S/SEAL/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "SEAL"
-version = v"3.5.5"
+version = v"3.5.6"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/microsoft/SEAL/archive/v3.5.5.tar.gz",
-                  "ae987c9df02a0a887eeb96032f307781b945183d3433e54a0a6d59207f4b8143"),
+    ArchiveSource("https://github.com/microsoft/SEAL/archive/v3.5.6.tar.gz",
```

Comment:
Pro tip:
```suggestion
    ArchiveSource("https://github.com/microsoft/SEAL/archive/v$(version).tar.gz",
```
so you don't have to manually update it :wink: